src/main/java/com/zaxxer/hikari/pool/PoolBagEntry.java

```diff
@@ -68,18 +68,22 @@
       DATE_FORMAT = new SimpleDateFormat("MMM dd, HH:mm:ss.SSS");
    }
 
-   public PoolBagEntry(final Connection connection, final HikariPool pool)
+   public PoolBagEntry(final Connection connection, final HikariPool pool, final boolean isReadOnly, final String catalog, final boolean isAutoCommit, final int networkTimeout, final int transactionIsolation)
```

Comment:
This is getting close to violating standards on the [number of parameters](http://checkstyle.sourceforge.net/config_sizes.html#ParameterNumber).